doc/man/man3/crypto_key_exchange.3monocypher

```diff
@@ -179,6 +180,14 @@ If either of the long-term secret keys leaks, it may compromise
 .Em all past messages .
 This can be avoided by using protocols that provide forward secrecy,
 such as the X3DH key agreement protocol.
+.Pp
+Many (private, public) key pairs produce the same shared secret.
+Therefore, not including the public keys in the key derivation can 
+lead to subtle vulnerabilities. This can be avoided by hashing the
+shared secret concatenated with both public keys. For example,
+.Fn BLAKE2b(shared_secret || your_pk || their_pk)
```

Comment:
[`.Fn`](https://man.openbsd.org/mdoc.7#Fn) is used only for function names. `BLAKE2b()` is not a function.

Instead use [`.Dl`](https://man.openbsd.org/mdoc.7#Dl) (indented) or [`.Ql`](https://man.openbsd.org/mdoc.7#Ql).